packages/components/select/__tests__/options.test.tsx

```diff
@@ -61,14 +61,11 @@ describe('options', () => {
     createWrapper({
       default: () =>
         samples.map((_, i) => (
-          <ElOptionGroupStub label={getLabel(i)}>
+          <ElOptionGroupStub>
             {{
               default: () =>
                 samples.map((_, j) => (
-                  <ElOptionStub
-                    label={getLabel(`${i}-${j}`)}
-                    value={j}
-                  ></ElOptionStub>
+                  <ElOptionStub label={getLabel(`${i}-${j}`)}></ElOptionStub>
                 )),
```

Comment:
Can you explain why this has been modified in this way?

Reply:
I was not aware that the type issue had been fixed by #17020. Will revert the code.